exasol_script_languages_container_ci/lib/ci.py

```diff
@@ -52,15 +53,16 @@ def ci(ctx: click.Context,
     rebuild = False
     push_to_public_cache = False
 
-    IS_REBUILD = re.compile(r"refs/heads/rebuild/.*")
-    IS_MASTER = re.compile(r"refs/heads/master")
+    IS_REBUILD_REGEX = re.compile(r"refs/heads/rebuild/.*")
```

Comment:
Should we maybe switch from rebuild branch prefix to commit message addition [rebuild], or use both

Reply:
Hm, personally I prefer the commit message variant. It produces less branches, and less costs

Reply:
Ok, I think the same, then let's add jt

Reply:
But maybe in another PR, this one is already large

Reply:
I created https://github.com/exasol/script-languages-container-ci/issues/11